classes/local/step/reader_json.php

```diff
@@ -179,7 +179,8 @@ public function form_add_custom_inputs(\MoodleQuickForm &$mform) {
             'modified' => [1654058940],
             'errors' => [],
         ];
-        $jsonexample = html_writer::empty_tag('br').html_writer::nonempty_tag('code', json_encode($arrayexample, JSON_PRETTY_PRINT));
+        $jsonexample = html_writer::empty_tag('br').
+            html_writer::nonempty_tag('pre', json_encode($arrayexample, JSON_PRETTY_PRINT));
```

Comment:
I would have personally preferred something like:
```suggestion
        $jsonexample = html_writer::empty_tag('br');
        $jsonexample .= html_writer::nonempty_tag('pre', json_encode($arrayexample, JSON_PRETTY_PRINT));
```
But this is fine for now.